.travis.yml

```diff
@@ -11,16 +10,17 @@ env:
     - CI=1
 
 rvm:
-  - 2.3.1
   - 2.4.2
+  - 2.6.3
 
 jdk:
-  - oraclejdk8
+  - oraclejdk9
 
 services:
   - redis-server
 
 before_install:
+  - rm -rf "${TRAVIS_BUILD_DIR}/tmp/solr-test"
```

Comment:
Where is the `TRAVIS_BUILD_DIR` defined?

Reply:
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables